uncontrast_studies/models/effect.py

```diff
@@ -0,0 +1,49 @@
+from django.db import models
+from django.db.models import CASCADE
+from simple_history.models import HistoricalRecords
+
+
+class UnConEffectOutcome(models.Model):
+    type = models.CharField(null=False, blank=False, max_length=250)
+
+    def __str__(self):
+        return self.type
+
+
+class UnConEffect(models.Model):
+    experiment = models.ForeignKey(
+        null=False, blank=False, to="uncontrast_studies.UnConExperiment", on_delete=CASCADE, related_name="findings"
+    )
+    outcome = models.ManyToManyField(null=False, blank=False, to=UnConEffectOutcome)
+    significance = models.BooleanField(null=False, blank=False)
```

Comment:
usually for booleans I prefer a relavant prefix: is, has, etc for example "is_significant"